### PR TITLE
Add own solution

### PR DESCRIPTION
**Pull request by Nevix123**
```
This is a PR template, If you are adding a solution link to the curriculum leave this as is. If you are not, then delete it and write the message you wish.
Thank you,
The Odin Project team.

```

### CODE REVIEW

File: web_development_101/project_html_css.md [hunk 1]
```diff
@@ -1271,6 +1271,9 @@ Submit a link below to the github repo with your files in it by using a pull req
 * [BlueSmitty's solution (Easy)](https://github.com/BlueSmitty/google-homepage) - [View in browser](https://bluesmitty.github.io/google-homepage/)
 * [Ehts2's sloution (Easy)](https://github.com/ehts2/googleHomepage) - [View in browser](https://ehts2.github.io/googleHomepage/)
 * [Dandala's solution (Easy)](https://github.com/Dandala88/google-search) - [View in browser](https://Dandala88.github.io/google-search/)
+* [Neven's solution (Easy)] (https://github.com/Nevix123/google_homepage) - [View in browser] (https://nevix123.github.io/google_homepage/)
```
Comment (mindovermiles262) [inline web_development_101/project_html_css.md]: Please remove the space between the closing `]` and the opening`(` of the links.  For example, `[<< link text here >>](<< url here >>)`

File: web_development_101/project_html_css.md [hunk 2]
```diff
@@ -1271,6 +1271,9 @@ Submit a link below to the github repo with your files in it by using a pull req
...
+* [Neven's solution (Difficult)] (https://github.com/Nevix123/google_first_page) - [View in browser] (https://nevix123.github.io/google_first_page/)
```
Comment (mindovermiles262) [inline web_development_101/project_html_css.md]: Same thing here

**Comment by mindovermiles262** [comment]
Hi @Nevix123 👋 

In order for Markdown links to work properly, there should not be a space between the closing `]` and opening `(`.  Please remove these spaces and I will merge this pull request.

If you have any questions or are unsure about how to edit this, please come say hello in our [odin chatroom](https://gitter.im/TheOdinProject/theodinproject) where someone will be available to help you

Thanks!

**Comment by Nevix123** [comment]
Hi, Andy,

Thanks  for replying.
I apologize for the mistake, I google-ed how to edit pull requests and hope it's ok now.

Let me know if there's something else that needs changing.

Thank you!

**Comment by mindovermiles262** [comment]
Looks good 👍

**Pull request merged**